packages/ui/primitives/document-flow/add-fields.tsx

```diff
@@ -653,6 +680,9 @@ export const AddFieldsFormPartial = ({
                       onMove={(options) => onFieldMove(options, index)}
                       onRemove={() => remove(index)}
                       onDuplicate={() => onFieldCopy(null, { duplicate: true })}
+                      onDuplicateAllPages={() =>
+                        onFieldCopy(null, { duplicate: true, duplicateAllPages: true })
```

Comment:
Is `duplicate: true` required here?

Reply:
good catch but yes, `duplicate: true` is required here,

currently this is the flow: 

if `duplicate` is false, it only copies the field to clipboard and returns early
if `duplicate` is true and `duplicateAllPages` is true, it creates copies across all pages
if `duplicate` is true and `duplicateAllPages` is false, it creates a single duplicate